server/constants/vars.js

```diff
@@ -11,7 +11,7 @@ export const SOCIODEMOGRAPHICS_FORM = 'form_sociodemographics'
 export const TOTAL_LABEL = 'Total'
 export const SITE = 'Site'
 
-export const ALL_FILTERS_ACTIVE = {
+export const ALL_FILTERS_INACTIVE = {
```

Comment:
should this have a `sites` key as well? I think it would help keep our code clean if we're always dealing with the same data shape.

Reply:
it has the sites key

Reply:
`DEFAULT_CHART_FILTERS` does but this one doesn't.

Reply:
ok so are you thinking of not having the sites be part of this filters object with the other filters in that case?

Reply:
yes, for this object i'm renaming the variable to accent that these are just the category filters
[3eddac5](https://github.com/AMP-SCZ/dpdash/pull/441/commits/3eddac54e283964e934cf306c476157501284b4f)